### PR TITLE
Solved responsive footer issue (email address and "contact us!" text). #61

### PR DESCRIPTION
**Pull request by Sunying-RONG**
for issue #61

- Make email address in footer responsive

- Make "Contact us!" text in button in footer responsive

Remind: As we use Jekyll, website rendering according to folder "_site", "_site" is generated automatically according to the files outside of "_includes" (e.g. If you only change the main.css in "_includes", but don't change the main.css in css folder outside of "_includes", the website will not show the changes).
Screenshot from 2019-07-06 20-38-45

**Before:**
Screenshot from 2019-07-06 11-45-29

**Now:**
Screenshot from 2019-07-06 20-16-10

### CODE REVIEW

File: _includes/css/main.css [hunk 1]
```diff
@@ -239,6 +239,23 @@
     #logo-footer-content span {
         font-size:6vw;
     }
+    #logo-footer-content a{
+        display:inline-block;
+        text-decoration:none;
+        background-color:#FF5658;
+        color:white;cursor:pointer;
```
Comment (rngadam) [inline _includes/css/main.css]: why is this on the same line?

File: _includes/css/main.css [hunk 2]
```diff
@@ -239,6 +239,23 @@
     #logo-footer-content span {
         font-size:6vw;
     }
+    #logo-footer-content a{
```
Comment (rngadam) [inline _includes/css/main.css]: spacing after a

File: _includes/css/main.css [hunk 3]
```diff
@@ -239,6 +239,23 @@
...
+        display:inline-block;
+        text-decoration:none;
+        background-color:#FF5658;
+        color:white;cursor:pointer;
+        font-family:Helvetica,Arial,sans-serif;
+        font-size:20px;
+        line-height:50px;
+        text-align:center;
+        margin:0;
```
Comment (rngadam) [inline _includes/css/main.css]: be consistent, one space after :

File: _includes/css/main.css [hunk 4]
```diff
@@ -501,13 +518,41 @@
...
+
     /* Section css */
     #section-info-container {
         position:relative;
         min-height:35%;
     }
 }
 /* css for big displays */
+@media only screen and (min-width:768px) and (max-width: 1080px){
+    #shanghai-franchise p a,
```
Comment (rngadam) [inline _includes/css/main.css]: not sure I understand why you need styling specific to each city?

**Comment by rngadam** [comment]
provide screenshot of output and a bullet-list of changes in your PR description

File: _includes/css/main.css (hunk 1 above)
Comment (Sunying-RONG) [reply _includes/css/main.css]: All these were in the same line in <a style="">, I copied them to main.css to do modification, and make it responsive. When I made them one by one in different lines, I missed this line, I changed now, put them in different lines.

File: _includes/css/main.css (hunk 2 above)
Comment (Sunying-RONG) [reply _includes/css/main.css]: Added the spaces

File: _includes/css/main.css (hunk 3 above)
Comment (Sunying-RONG) [reply _includes/css/main.css]: OK

File: _includes/css/main.css (hunk 4 above)
Comment (Sunying-RONG) [reply _includes/css/main.css]: Oh, this is written by others before, I haven't noticed that. Yes, the css code of three cities can be put together, no need to repeat. I changed it.

**Comment by rngadam** [comment]
Run a linter on the CSS code and fix error and create new issues for each category of problem: 

http://csslint.net

Separate all CSS blocks by one blank line.

Make linting part of the build process.

**Pull request merged**